intro-Linux/ch_hpc_infrastructure.tex

```diff
@@ -0,0 +1,87 @@
+\chapter{More on the HPC infrastructure}
+
+\section{Filesystems}
+
+In this section, we briefly explain which different filesystems are available on
+the \gls{HPC} infrastructure.
```

Comment:
We don't explain anything here...

"Multiple different shared filesystems are available on the HPC infrastructure, each with the own purpose."

then refer to other section